test/createAzureResource.test.ts

```diff
@@ -58,14 +59,39 @@ suite('Create Azure Resources', async function (this: ISuiteCallbackContext): Pr
         ext.tree.dispose();
     });
 
-    test('createFunctionApp (Basic)', async () => {
+    test('Create windows function app (Basic) and deploy JavaScript project', async () => {
```

Comment:
Make `CreateFunctionApp` and `Deploy Javascript project` tests two separate tests.